### PR TITLE
[#185] WIP start documenting Symfony toggle

### PR DESCRIPTION
**Pull request by kpicaza**
The first iteration on issue #185, documenting only in-memory config details.

- [x] default config
- [x] in-memory config
- [ ] dbal config
- [ ] http-api config

### CODE REVIEW

**Comment by kpicaza** [comment]
I will continue documenting Dbal and CRUD since I continue implementing example app https://github.com/kpicaza/pheature-symfony-pre-example

File: docs/getting-started/symfony-bundle.md [hunk 1]
```diff
@@ -12,47 +12,52 @@ Install it using [composer package manager](https://getcomposer.org/download/) i
...
-  Pheature\Sdk\CommandRunner:
-    class: Pheature\Sdk\CommandRunner
-    arguments:
-      - '@Pheature\Core\Toggle\Read\Toggle'
+pheature_flags:
+  driver:               ~ # One of "inmemory"; "dbal"
+  prefix:               ''
+  segment_types:
+    - { type: 'identity_segment', factory_id: 'Pheature\Model\Toggle\SegmentFactory' }
+    - { type: 'strict_matching_segment', factory_id: 'Pheature\Model\Toggle\SegmentFactory' }
```
Comment (xserrat) [inline docs/getting-started/symfony-bundle.md:38]: Here we could also add the `in_collection_matching_segment`. What do you think?

**Review by xserrat — APPROVED** [review]
It looks really good! 👏🏽 🚀

File: docs/getting-started/symfony-bundle.md (hunk 1 above)
Comment (kpicaza) [reply docs/getting-started/symfony-bundle.md:38]: Is a nice point;-D, we will create a new issue to add the `in_collection_matching_segment` to the default loaded Segments in Symfony config.
image

When we make the bundle it doesn't already exist.

**Pull request merged**